home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/service/impl/UserServiceImpl.java

```diff
@@ -38,9 +39,7 @@
     @Override
     @Transactional
     public UserDto createUser(UserDto createUserDto) {
-        if (userRepository.findByEmail(createUserDto.getEmail()).isPresent()) {
-            throw new AlreadyExistHomeException("User with email" + createUserDto.getEmail() + " is already exists");
-        } else {
+        if (Boolean.FALSE.equals(userRepository.findByEmail(createUserDto.getEmail()).isPresent())) {
```

Comment:
🥺
![image](https://user-images.githubusercontent.com/19805592/116544347-af843e00-a8f7-11eb-956a-b848d0f7dca7.png)

Reply:
it was double check))